de.metas.migration/de.metas.migration.cli/RolloutMigrate.launch

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>

```

Comment:
Yes, you are right. Documentation is good in any case.
Where is the best place? 
- README_rolloutmigrate.md at the same place where the launcher is located as you suggest. The logical consequence woud be some readme in `de.metas.endcustomer.mf15.swingui` for the launchers defined there
- http://docs.metasfresh.org/pages/developers/index_en resp. 
- http://docs.metasfresh.org/developers_collection/en/getting_started_ide_en.html where other eclipse launcher are mentioned 
- or any other place

Anyway, I suggest 
- I write some sentences here (**in the pull requet**) as comments
- you decide where to place them (it is md, so you can copy&paste it)

Its ok for you? What do you think?